requirements/default.txt

```diff
@@ -1,4 +1,5 @@
 pytz>dev
 billiard>=3.6.3.0,<4.0
+future>=0.18.0
```

Comment:
should this also include setup requirements?

Reply:
I don't think it's needed there. It's used only in celery.backends.base

Reply:
It is needed in `setup.py`, otherwise is doesn't get installed as a dependency of this package. See https://github.com/celery/celery/issues/6148